### ISSUE TITLE
Date keeps showing in IST format. Is there any support for UTC format.How to use utc-mode option.

### ISSUE THREAD

**Issue by ashu0992sharma** [zhaber/angular-js-bootstrap-datetimepicker#65]
I am editing a form. I have the date stored in utc format but datepicker changes it to IST format which gives me wrong time. Can you provide me any example or direct me in the right way.

**Comment by zhaber** [comment]
See https://github.com/zhaber/angular-js-bootstrap-datetimepicker/pull/37 This pull request is pending on addressing the comments, please feel free to follow up on it so this change can be merged into the master branch.
